.releaserc

```diff
@@ -8,9 +8,14 @@
   ],
   "plugins": [
     [
-      "@semantic-release/commit-analyzer", {
+      "@semantic-release/commit-analyzer",
+      {
         "releaseRules": [
-          { "type": "build", "scope": "Needs bump", "release": "patch" }
+          {
```

Comment:
What formatting options are applied here? IntelliJ IDEA does not do this

Reply:
I am using VSCode, as it's essentially a JSON file I don't need to use a heavy editor

Reply:
The reaso I am commenting on this is because I was intentionally keeping these files in sync with other of our repos, so if we make such changes I would like them to be synced to all our repos. Probably a formatted config should've been added.